bin/amphtml-update.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+# Go to the right location.
+if [[ '.' != $(dirname "$0") ]]; then
+	cd bin
+fi
+
+BIN_PATH="$(pwd)"
+PROJECT_PATH=$(dirname $PWD)
+VENDOR_PATH=$PROJECT_PATH/vendor
+
+if ! apt-get > /dev/null; then
+	echo -e "The AMP HTML uses the apt-get, make sure to run this script in a Linux environment"
+	exit
```

Comment:
Do `exit 1` so a failure error code is sent. An `exit` means just `exit 0` which means success.